grunt-configs/copy.js

```diff
@@ -20,6 +20,16 @@ module.exports = function (grunt, options) {
                 },
                 {
                     expand: true,
+                    cwd: 'node_modules',
+                    src: [
+                        'react/dist/react-with-addons.js',
+                        'JSON/json2.js'
+                    ],
```

Comment:
yeah very happy to do that in subsequent PRs, was trying to limit this to just changes that fell out of the first bower going